lib/hanami/http/status.rb

```diff
@@ -27,19 +27,24 @@ class Status
       # @example
       #   require 'hanami/http/status'
       #
-      #   Hanami::Http::Status.for_code(418) # => [418, "I'm a teapot"]
+      #   Hanami::Http::Status.for_code(409) # => [409, "Conflict"]
       def self.for_code(code)
         ALL.assoc(code)
       end
 
-      # Return a message for the given status code
+      # Returns a message for the given status code
       #
       # @param code [Integer] a valid HTTP code
       #
       # @return [String] a message for the given status code
       #
       # @since 0.3.2
       # @api private
+      #
+      # @example
+      #   require 'hanami/http/status'
```

Comment:
```suggestion
      #   require "hanami/http/status"
```

Reply:
Oh, right, I haven't checked for single quotes in code examples. Probably there's more of these (for example few lines above, from which I copied this example). Should I review it as well before this is merged?